server/utils/utils.ts

```diff
@@ -94,3 +94,24 @@ export const formatMoney = (val: number, emptyState: string = undefined, currenc
 export const formatPrivilegedVisitsSummary = (count: number): string => {
   return `Including ${count} privileged visits`
 }
+
+export const arrayToQueryString = (array: string[] | number[] | boolean[], key: string): string =>
+  array && array.map(item => `${key}=${encodeURIComponent(item)}`).join('&')
+
+export const mapToQueryString = (params: Record<never, never>): string =>
+  Object.keys(params)
+    .filter(key => params[key])
+    .map(key => {
+      if (Array.isArray(params[key])) return arrayToQueryString(params[key], key)
+      return `${key}=${encodeURIComponent(params[key])}`
+    })
+    .join('&')
+
+export const getNamesFromString = (string: string): string[] =>
+  string &&
+  string
+    .split(', ')
+    .reverse()
+    .join(' ')
+    .split(' ')
+    .map(name => properCaseName(name))
```

Comment:
This seems like it specifically needs a particular input string rather than just "get names from a string" so i wonder if it belongs in utils or if it could be just a local function to the overview page method.

Whether we keep it here or not, it'd probably be good to get some tests that test this particular bit of logic though as it seems important